pkg/kubectl/bind/plugin/bind.go

```diff
@@ -107,6 +108,18 @@ func (b *BindOptions) Complete(args []string) error {
 	if len(args) > 0 {
 		b.URL = args[0]
 	}
+
+	if b.DryRun {
+		b.PrintFlags.Complete("%s (dry run)") // nolint: errcheck
+	}
+
+	printer, err := b.PrintFlags.ToPrinter()
+	if err != nil {
+		return err
+	}
+
+	b.Printer = printer
```

Comment:
this is misplaced in the options IMO. Should be called further down in Run.

Reply:
Or does ToPrinter include validation? Then we should call it in the validate func.

Reply:
ToPrinter includes validation but it can not be in validate function. Because flag -> options mapping is done in complete. The cases validation and mapping intertwining mostly done in complete in kubectl.

Reply:
not sure I can follow.